src/sentry/static/sentry/app/components/search/sources/commandSource.jsx

```diff
@@ -25,6 +26,13 @@ const ACTIONS = [
       }),
   },
 
+  {
+    title: 'Toggle Translation Markers',
+    description: 'Toggles translation markers on or off in the application',
```

Comment:
Ironically this should be translated, but the others aren't, so we'll come back for these later.